test-assignments/frontend-react-assignment/README.md

```diff
@@ -16,14 +16,16 @@ Please submit the base project as your initial commit and implement the tasks in
 
 - Create a new component which does a basic pagination.
   - This component should contain 2 buttons (Prev, Next) which can paginate the colleague list in the correct direction. The list should only show 5 elements at a time.
-  - Hint: Try to use a Higher Order Component. Do NOT use a 3rd party library for the pagination feature itself but feel free to use any other package for the implementation.
```

Comment:
HOCs are not as common as their used to be. The assignment can still implement with a HOC, but it's more common to use custom hooks now, so I reworded this section to highlight the reusability, but let the candidate decide how he implements it